README.md

```diff
@@ -51,6 +51,16 @@ Following ENV variables can be set:
     setting it to anything else. LOG_MODE defaults to only "errors".
 -   `SAS_SUBSTRATE_WS_URL`: WebSocket URL to which the RPC proxy will attempt to connect to, defaults to
     `ws://127.0.0.1:9944`.
+-   `SAS_SUBSTRATE_PARENT_VERSION`: always use the version of the parent block to set
+    metadata for scale decoding. `on` means always fetch and use the parent block's metadata, while `off`
+    means _only_ fetch and use the parent blocks metadata for the blocks with hashes specified in
+    `config/setCodeBlocks.ts`. Fetching and using the parent block's metadata will have a slight time
+    penalty, but will guarantee that extrinsics within a block that executed a runtime upgrade with
+    `set_code` are always decoded correctly. Defaults to `off`. Note: for chains that use the
+    `scheduler` pallet to do runtime upgrades, it is very unlikely this will be an issue as
+    `scheduler` ensures that `set_code` is executed in `on_initialize` and it is highly unlikely
+    there would be any additional room in a block for other extrinsics, but please use this info with
+    discretion.
```

Comment:
Might try to make this a bit more helpful/clear by updating to something like...

```suggestion
    `set_code` are always decoded correctly. Defaults to `off`. We recommend that chains that
    use scheduled upgrades use the default behavior, however, please consider your use case
    when making a final decision.
```